Enterprise-CMCS/managed-care-review
# MCR-3772 MCR-3771 Edit and submit standalone rate

## Summary
- State can submit an edited rate from rate form and is routed to dashboard where edits have been applied
- State can edit a rate form for a single unlocked rate

🔍 Most important changes here are to the new files in the `V2` folder. That is the new rate details page and standalone form.

#### Related issues
https://jiraent.cms.gov/browse/MCR-3772
https://jiraent.cms.gov/browse/MCR-3771

#### Test cases covered
No tests implemented. We are not turning this on for awhile. I moved testing out this ticket and into new [ticket](https://jiraent.cms.gov/browse/MCR-3933).

## QA guidance

Please run through the manual QA steps on these tickets and check the acceptance criteria. No automated tests yet.
